CIP-1694/README.md

```diff
@@ -1053,9 +1071,17 @@ Secondly, during the bootstrap phase, a vote from the constitutional committee,
 together with a sufficient SPO vote, is sufficient to initiate a hard fork.
 No other actions are possible during the bootstrap phase.
 
-The bootstrap phase ends when **either** of the following occurs:
-1. a sufficient amount of stake is delegated to DReps
-2. a given number of epochs has passed (likely to be a number of months after the hard fork)
+The bootstrap phase ends when a given number of epochs has elapsed,
+as specified in the next ledger era configuration file.
+This is likely to be a number of months after the hard fork.
+
+Moreover, there will be an interim Constitutional committee,
+also specified in the next ledger era configuration file,
+whose term limits will be set to expire when the bootstrap phase end.
+The interim committee will be responsible for electing the first non-bootstrap committee.
```

Comment:
This makes it sound like the interim committee just chooses the first regular committee.

Reply:
Good point. I think I can just delete this line altogether, since this would all be social anyway (they have no voting power on the non-bootstrap committee).